wordpress/dto/WordpressUserProfileListDTO.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { WordpressUserProfileDTO } from "./WordpressUserProfileDTO";
+import { LogService } from "../../LogService";
+
+const LOG = LogService.createLogger('Wordpress-Userprofiles');
+
+
+export type WordpressUserProfileListDTO = WordpressUserProfileDTO[];
+
+export function isWordpressUserProfilesDTO (value: any): value is WordpressUserProfileListDTO {
+    LOG.debug('reached')
```

Comment:
This clearly is just for debugging... but  It's bad practice to log from this type of function that may be called a lot.